packages/wonder-blocks-timing/src/util/types.ts

```diff
@@ -18,7 +18,7 @@ export interface ITimeout {
      * false.
      * @memberof ITimeout
      */
-    isSet(): boolean;
+    get isSet(): boolean;
```

Comment:
This probably isn't common enough to update the codemod, but maybe worth mentioning in the upgrade guide?

Reply:
I was planning on fixing it before converting Perseus.  I don't think it'll take long to do.